consensus/polybft/wallet/key.go

```diff
@@ -23,8 +28,8 @@ func (k *Key) Address() ethgo.Address {
 	return k.raw.Ecdsa.Address()
 }
 
-func (k *Key) NodeID() pbft.NodeID {
-	return pbft.NodeID(k.String())
+func (k *Key) NodeID() string {
+	return k.String()
 }
```

Comment:
I would remove `NodeID` function completely and use `String` instead.

Reply:
https://github.com/0xPolygon/polygon-edge/pull/865/commits/e9f82c8aa934344e1b727726a10daa473becae37